python_modules/dagster/dagster/core/definitions/job.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         preset_defs: Optional[List[PresetDefinition]] = None,
         tags: Dict[str, Any] = None,
         hook_defs: Optional[AbstractSet[HookDefinition]] = None,
-        solid_retry_policy: Optional[RetryPolicy] = None,
+        retry_policy: Optional[RetryPolicy] = None,
```

Comment:
how do people feel about renaming it without going through a deprecation cycle? 
we don't recommend using `JobDefinition` a lot, so i kinda feel it's ok to just rename it.

Reply:
I kinda feel like it's also okay to break this.

But I'm more curious about the name change from `solid_retry_policy` => `retry_policy`, and whether the wordiness is actually worth it because it's unambiguous what it's referring to.  Also, is it valuable to keep Pipeline and Job have similar-ish arguments.

Reply:
yea I can imagine a world where a `job_retry_policy` exists that is separate from the `op_retry_policy`

Reply:
op_retry_policy sounds good, and related to that should hook_defs be op_hook_defs then?